### PR TITLE
Bump prices dependencies

### LINKED ISSUE #22: Bump prices to 1.0.0-beta
This library should be adjusted to work with the new [prices](https://github.com/mirumee/prices) API.

### PR DESCRIPTION
**Pull request by koradon**
Fixes #22

### CODE REVIEW

File: django_prices_openexchangerates/templatetags/prices_multicurrency.py [hunk 1]
```diff
@@ -8,20 +9,23 @@
 
 @register.simple_tag
 def gross_in_currency(price, currency, **kwargs):
```
Comment (patrys) [inline django_prices_openexchangerates/templatetags/prices_multicurrency.py]: This no longer makes sense. You can do `{{ price.gross|in_currency:'USD' }}`. What's the point of converting both net and gross and then immediately discarding one of them?

File: setup.py [hunk 2]
```diff
@@ -34,9 +34,9 @@
         'django_prices_openexchangerates.templatetags'],
     include_package_data=True,
     classifiers=CLASSIFIERS,
-    install_requires=['Django>=1.4', 'django-prices>=0.6.1', 'prices>=0.5.2'],
+    install_requires=['Django>=1.4', 'pytest-runner', 'django-prices>=0.6.1', 'prices>=1.0.0-beta'],
```
Comment (patrys) [inline setup.py]: What is `pytest-runner` doing here?

File: django_prices_openexchangerates/tests.py [hunk 3]
```diff
@@ -28,47 +28,27 @@ def mock_rates(currency):
...
-        self.assertEqual(first_operation.left, base_price)
-        self.assertEqual(first_operation.right.base_currency, 'EUR')
-        self.assertEqual(first_operation.right.to_currency, 'USD')
-        second_operation = converted_price.history.right
-        self.assertEqual(second_operation.base_currency, 'USD')
-        self.assertEqual(second_operation.to_currency, 'GBP')
 
     def test_convert_price_uses_passed_dict(self, mock_qs):
-        base_price = Price(10, currency='USD')
+        base_price = Amount(10, currency='USD')
```
Comment (patrys) [inline django_prices_openexchangerates/tests.py]: This name is misleading as it's not a Price instance.

File: setup.py (hunk 2 above)
Comment (koradon) [reply setup.py]: This is still in progress, with @elwoodxblues we had a problem to run tests. For my environment it is still not resolved. Adding pytest-runner solved problem for @elwoodxblues. I am investigating this problem right now.

File: django_prices_openexchangerates/templatetags/prices_multicurrency.py (hunk 1 above)
Comment (koradon) [reply django_prices_openexchangerates/templatetags/prices_multicurrency.py]: This PR is still in progress. For now tests are updated to pass.

File: django_prices_openexchangerates/tests.py (hunk 3 above)
Comment (koradon) [reply django_prices_openexchangerates/tests.py]: Yes, i will update tests shortly.

File: setup.py (hunk 2 above)
Comment (elwoodxblues) [reply setup.py]: We were checking if `pytest-runner` would help us locally, but this change shouldn't be committed.

Comment (koradon) [reply setup.py]: ok i will delete this

**Comment by elwoodxblues** [comment]
@koradon @patrys Tests in this package are written using `unittest`, which is not our standard tool. Shouldn't we use `pytest` instead? Also, it would be great if we had the same testing stack on CI. Recently we've changed `django-prices` and `django-payments` to use `tox` and `tox-travis`. Unifying the supported Python and Django version in all packages would also be welcomed.

**Comment by elwoodxblues** [comment]
@koradon We should respectively bump the version of this package as well.

**Comment by patrys** [comment]
@elwoodxblues 👍  I'd expect all our packages to use a similar stack for testing.

**Comment by koradon** [comment]
I'm on it

**Comment by koradon** [comment]
@elwoodxblues @patrys pls review
it should be django 2.0 compatible (checked locally)

**Pull request merged**